packages/kit/src/exports/node/index.js

```diff
@@ -59,7 +58,13 @@ function get_raw_body(req, body_size_limit) {
 
 				size += chunk.length;
 				if (size > length) {
-					controller.error(new Error('content-length exceeded'));
+					req.destroy(
+						new Error(
+							`request body size exceeded ${
+								content_length ? "'content-length'" : 'BODY_SIZE_LIMIT'
+							}: ${length}`
```

Comment:
one last suggestion. it might be slightly unclear which length is being printed, so I think this small wording change would help
```suggestion
							} of ${length}`
```